src/solver/type_getters.jl

```diff
@@ -520,8 +520,10 @@ function get_callbacks(parsed_args, simulation, atmos, params)
             call_every_n_steps(turb_conv_affect_filter!; skip_first = true),
         )
     end
-
-    return ODE.CallbackSet(callbacks...)
+    return ODE.CallbackSet(
```

Comment:
This callback should probably be conditionally added, like the others. Sometimes we don’t have radiation.